Tribler/community/multichain/statistics/page_rank.py

```diff
@@ -0,0 +1,74 @@
+from random import random, choice
+
+
+class IncrementalPageRank(object):
+    def __init__(self, graph, walks_per_node=2, stop_probability=0.1):
```

Comment:
Looks like some random numbers here. Is this the right place for them? Maybe put them in a .config or so?